### PR TITLE
Added Letsencrypt support to docker-mailserver

### LINKED ISSUE #35: Easy install and usage of LetsEncrypt certificates
This has been discussed [here](https://github.com/tomav/docker-mailserver/issues/33) and could be started in December when LetsEncrypt beta will go Public.

### PR DESCRIPTION
**Pull request by tomav**
Added Letsencrypt support to docker-mailserver, fixes #35

### CODE REVIEW

**Comment by tomav** [comment]
You can try the `letsencrypt` version, just `docker pull tvial/docker-mailserver:letsencrypt ` and read `SSL.md` in the `letsencrypt` branch.

File: start-mailserver.sh [hunk 1]
```diff
@@ -54,28 +54,50 @@ echo "Postfix configurations"
...
+case $DMS_SSL in
+  "letsencrypt" )
+    # letsencrypt folders and files mounted in /etc/letsencrypt
+
+      # Postfix configuration
+      sed -i -r 's/smtpd_tls_cert_file=\/etc\/ssl\/certs\/ssl-cert-snakeoil.pem/smtpd_tls_cert_file=\/etc\/letsencrypt\/live\/'$(hostname)'\/fullchain.pem/g' /etc/postfix/main.cf
+      sed -i -r 's/smtpd_tls_key_file=\/etc\/ssl\/private\/ssl-cert-snakeoil.key/smtpd_tls_key_file=\/etc\/letsencrypt\/live\/'$(hostname)'\/privkey.pem/g' /etc/postfix/main.cf
+
+      # Courier configuration
+      cat /etc/letsencrypt/live/$(hostname)/privkey.pem /etc/letsencrypt/live/$(hostname)/cert.pem >> /etc/letsencrypt/live/$(hostname)/combined.pem
```
Comment (jderusse) [inline start-mailserver.sh]: should be `>` 
unless yout want to append the 2 certificats into the shared file `combined.pem` each time you call this script ;)

Comment (tomav) [inline start-mailserver.sh]: Hi @jderusse, you're right, it's an error. I have it in another script too. I'm gonna push the fix.

**Comment by tomav** [comment]
I'll merge on Monday.

**Pull request merged**